Include/Beman/InplaceVector/inplace_vector.hpp

```diff
@@ -191,13 +192,13 @@ struct inplace_vector_base : public inplace_vector_destruct_base<T, Capacity> {
   template <typename Iter>
   static constexpr void uninitialized_copy(Iter first, Iter last,
                                            iterator dest) noexcept {
```

Comment:
I think we need conditional noexcept here

Reply:
So here I need to consider the cases where it is guaranteed that the function doesn't throw any exception. Since it does copying - I think a traditional 
```cpp
noexcept(std::is_nothrow_copy_constructible_v<typename std::iterator_traits<Iter>::value_type>)
```
should suffice.
@JeffGarland

Reply:
That looks correct to me.